project.mml

```diff
@@ -285,14 +285,22 @@ Layer:
       <<: *osm2pgsql
       table: |-
         (SELECT
-             way, "natural"
-           FROM planet_osm_polygon
-           WHERE "natural" IN ('spring')
-         UNION ALL
-         SELECT
-             way, "natural"
-           FROM planet_osm_point
-           WHERE "natural" IN ('spring')
+            way,
+            "natural"
+          FROM
+          (SELECT
+              ST_PointOnSurface(way) AS way,
+              "natural"
+            FROM planet_osm_polygon
+            WHERE way && !bbox!
+          UNION ALL
+          SELECT
+              way,
+              "natural"
+            FROM planet_osm_point
+            WHERE way && !bbox!
+            ) _
```

Comment:
Looks like unwanted space + underscore.

Reply:
What would you suggest as an alternative for what to call the inner subselect? `_` is a reasonably standard name when you just need *a* name. `s`, `p`, `q` are also common throw-away names.

Reply:
I don't know, however I don't like such shortcuts in the big SQL section of a public project. Typical `i` for loop index might be OK in typical code, but this one is already hard to maintain, underscore is worse than any letter and really looks like a typo, and you can't expect that other people will be real programmers to catch it. Something self-commenting like `inner_subselect` would be probably good.

BTW: there are more undocumented conventions and sections that I don't understand, it might make sense to add more comments one day.

Reply:
`inner_subselect` as a name would do for me.